src/video/glutil.hpp

```diff
@@ -43,7 +43,7 @@ using namespace gl;
 #  include <GLES/glext.h>
 #else
 #  include <GL/gl.h>
-#  include <GL/glext.h>
+//#  include <GL/glext.h>
```

Comment:
Why did you remove this include? Please revert it, I don't know what side effects this might have.